### ISSUE TITLE
Bulk pubsub documentation does not list supported components

### ISSUE THREAD

**Issue by kayvonkhosrowpour** [dapr/docs#3587]
**What content needs to be created or modified?**
The bulk pubsub documentation page doesn't include a list of the supported components. There is a comment:

> Please refer [Supported components](https://docs.dapr.io/developing-applications/building-blocks/pubsub/pubsub-bulk/).

But it links to the page itself, which doesn't contain the list of supported components.

**Describe the solution you'd like**
A list of the supported components, and an indication of what might happen if you try a bulk subscription on an unsupported component.

**Where should the new material be placed?**
At the top of the bulk pubsub doc page, under the intro section. 

**The associated pull request from dapr/dapr, dapr/components-contrib, or other Dapr code repos**
None

**Additional context**
I am not sure what components are supported, but interested in knowing more.

**Comment by shubham1172** [comment]
@kayvonkhosrowpour hi, all components support bulk pub/sub today, that comment should be removed.

However, some components are more efficient at bulk pub/sub than the others (and that is an implementation detail). @yaron2 and @msfussell, since we removed the columns here https://github.com/dapr/docs/pull/3179, what do you suggest as a recommended way for this? I think it is valuable for end-users to know which components do a better job to help them choose one for their use case, so maybe we could word it differently? Instead of saying XYZ supports bulk pub/sub (which is untrue because all components support it), could we instead say XYZ is optimized for bulk pub or bulk sub? 

/cc @DeepanshuA @mukundansundar

**Comment by msfussell** [comment]
@shubham1172 - I do not see the value in saying some are more performant than others, unless there is a real reason to draw this out. We could say the same for state management or pub/sub components in general. The best way to address this is to have a perf test suite that can be run that shows the numbers.

**Comment by shubham1172** [comment]
> unless there is a real reason to draw this out

I believe there is, because of the way it's implemented. We have done extra work for some components to make them much faster than the other components. IMO there is value for users to know which components are better and value for developers to identify which components need "native implementation" to make them further better. 

> The best way to address this is to have a perf test suite that can be run that shows the numbers.

Perf tests are good, but in this case we are not comparing two components X and Y on their actual performance. All we are saying is that for X, we send messages from Dapr to X in a bulk request, whereas for Y, we send messages from Dapr to Y in parallel single requests.

**Comment by DeepanshuA** [comment]
> I do not see the value in saying some are more performant than others

I beg to differ on this. The entirety of work that has been done for bulk-pubsub matters to all components BUT it matters in a major way for those components which support it natively.
So, as a user, I should have this visibility, so that I am able to make an intelligent decision.

Though it is arguable, but I still think that one of the mistakes that we made was not having a dedicated word to be able to differentiate between these two categories - we could have used something like `natively`. Or some other word but there should be one to differentiate that.

Perf tests can certainly help but still as a developer trying to make intelligent decisions, I should be able to get this piece of information somehow somewhere that whether messages from Dapr to a particular component are going in a bulk manner or just parallel requests.

**Comment by msfussell** [comment]
Can we add the list of brokers that support bulk-pub/sub directly? Then we can add this into the docs.

**Comment by mukundansundar** [comment]
Duplicate of #3748

**Comment by msfussell** [comment]
See these links from the duplicate
-  https://github.com/dapr/docs/issues/3748#issuecomment-1734083351
- https://github.com/dapr/docs/issues/3748#issuecomment-1746247842

**Comment by hhunter-ms** [comment]
> Duplicate of #3748

Closing this in favor of 3748 - feel free to reopen if necessary
